src/utils.js

```diff
@@ -74,9 +69,30 @@ function toClassName(text) {
     .replace(/[^0-9a-z]/gi, '-');
 }
 
+/**
+ * Adds the fastly-image-optimization url params to the given image src.
+ * @param {string} src The image source.
+ * @param {number} [width = 0] optional 'width' parameter
+ * @param {string} [format = 'webply'] image format.
+ * @param {string} [optimize = 'medium'] optimization.
+ * @returns {string}
+ */
+function optimizeImageURL(src, width, format = 'webply', optimize = 'medium') {
```

Comment:
Should we go for a `optimizeImageURL(src, options)` signature instead to allow overriding just a single param and for future extensions?

Reply:
hmm... I thought about it, and I thought it's overkill. we can still add it, if we need it.